File: SEIST/polyglot/swift/SEISMotionPolicy.swift
import Foundation

struct SEISMotionPolicy {
    let supportsReducedMotion: Bool
    let mobileParticleLimit: Int
    let desktopParticleLimit: Int
    let continuousSceneLimit: Int

    static let foundation = SEISMotionPolicy(
        supportsReducedMotion: true,
        mobileParticleLimit: 24,
        desktopParticleLimit: 64,
        continuousSceneLimit: 1
    )
}

File: SEIST/polyglot/dart/seis_motion_policy.dart
class SeisMotionPolicy {
  const SeisMotionPolicy({
    required this.supportsReducedMotion,
    required this.mobileParticleLimit,
    required this.desktopParticleLimit,
    required this.continuousSceneLimit,
  });

  final bool supportsReducedMotion;
  final int mobileParticleLimit;
  final int desktopParticleLimit;
  final int continuousSceneLimit;

  static const foundation = SeisMotionPolicy(
    supportsReducedMotion: true,
    mobileParticleLimit: 24,
    desktopParticleLimit: 64,
    continuousSceneLimit: 1,
  );
}

